examples/kubernetes/cluster/main.tf

```diff
@@ -13,12 +9,12 @@ resource "vkcs_kubernetes_cluster" "k8s-cluster" {
   }
 
   name                = "k8s-cluster"
-  cluster_template_id = data.vkcs_kubernetes_clustertemplate.ct.id
-  master_flavor       = data.vkcs_compute_flavor.k8s.id
+  cluster_template_id = data.vkcs_kubernetes_clustertemplate.example_template_by_version.id
+  master_flavor       = data.vkcs_compute_flavor.basic.id
   master_count        = 1
 
-  network_id          = vkcs_networking_network.k8s.id
-  subnet_id           = vkcs_networking_subnet.k8s-subnetwork.id
+  network_id          = vkcs_networking_network.app.id
+  subnet_id           = vkcs_networking_subnet.app.id
```

Comment:
may be remove it?

Reply:
we cannot remove subnet.id 
without it there is error


"""
│ Error: Missing required argument
│
│   on main.tf line 1, in resource "vkcs_kubernetes_cluster" "k8s-cluster":
│    1: resource "vkcs_kubernetes_cluster" "k8s-cluster" {
│
│ The argument "subnet_id" is required, but no definition was found.
"""